pkg/plugin/packetparser/_cprog/packetparser.c

```diff
@@ -5,39 +5,40 @@
 #include "bpf_helpers.h"
 #include "bpf_endian.h"
 #include "packetparser.h"
+#include "conntrack.c"
+#include "conntrack.h"
 #include "retina_filter.c"
 #include "dynamic.h"
 
 char __license[] SEC("license") = "Dual MIT/GPL";
 
-
-
 struct tcpmetadata {
 	__u32 seq; // TCP sequence number
 	__u32 ack_num; // TCP ack number
+	__u32 tsval; // TCP timestamp value
+	__u32 tsecr; // TCP timestamp echo reply
```

Comment:
move these fields to the beginning so that you can do a block memory copy when assigning values to this type in `parse()` at `packetparser.c@205`